vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,272 @@
+
+gmps:preferrer-tenderer a skos:Concept ;
+    skos:definition "The applicant is the preferred tenderer."@en ;
+    skos:broader gmps:application,
+        gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Preferred Tenderer"@en .
+
+gmps:priority-applicant a skos:Concept ;
+    skos:definition "The applicant is a Priority Applicant for competing application."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Priority Applicant"@en .
+
+gmps:ranked a skos:Concept ;
+    skos:definition "The application for the Resource Permit is a Ranked Application, pending grant of a Priority Application."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Ranked"@en .
+
+gmps:rejected a skos:Concept ;
+    skos:definition "The Resource Permit has been rejected."@en ;
+    skos:broader gmps:non-current,
+        gmps:not-accepted,
+        gmps:prelodgement ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Rejected"@en .
+
+gmps:renewal-application-rejected a skos:Concept ;
+    skos:definition "The Renewal Application for the Resource Permit has been rejected."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Renewal Application rejected"@en .
+
+gmps:renewal-lodged a skos:Concept ;
+    skos:definition "The Application of Renewal for the Resource Permit has been lodged."@en ;
+    skos:broader gmps:granted,
+        gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Renewal lodged"@en .
+
+gmps:renewal-withdrawn a skos:Concept ;
+    skos:definition "The Application of Renewal for the Resource Permit has been withdrawn."@en ;
+    skos:broader gmps:granted,
+        gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Renewal withdrawn"@en .
+
+gmps:replacement-permit a skos:Concept ;
+    skos:definition "The permit represents a Replacement Permit."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Replacement Permit"@en .
+
+gmps:replacement-permit-application a skos:Concept ;
+    skos:definition "The application represents a Replacement Permit Application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Repalcement Permit Application"@en .
+
+gmps:sub-blocks-all-relinquished a skos:Concept ;
+    skos:definition "All sub-blocks have been relinquished."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Sub-blocks all relinquished"@en .
+
+gmps:surrendered a skos:Concept ;
+    skos:definition "The Resource Permit has been surrendered."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Surrendered"@en .
+
+gmps:unfinalised a skos:Concept ;
```

Comment:
What is unfinalised? How is it different to abandoned or surrendered or sub-blocks-all-relinquished?